test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Resources/ApplicationConsumingPrecompiledViews.Manage.Home.Index.txt

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <body>
-        AspNetCore.Areas_Manage_Views_Shared__Layout_cshtml, ClassLibraryWithPrecompiledViews.PrecompiledViews, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
+        AspNetCore._Areas_Manage_Views_Shared__Layout_cshtml, ClassLibraryWithPrecompiledViews.PrecompiledViews, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
```

Comment:
We changed how the paths were being calculated - instead of using the ones from `IFileProvider`, we calculate it from the absolute paths. This has the slightly intended consequence of fixing https://github.com/aspnet/MvcPrecompilation/issues/27